step-capstone/src/components/TimeLine.js

```diff
@@ -0,0 +1,166 @@
+import React, {useState, useEffect} from 'react';
+import '../styles/TimeLine.css';
+import DateFnsUtils from '@date-io/date-fns';
+import {
+ MuiPickersUtilsProvider,
+ KeyboardDatePicker,
+} from '@material-ui/pickers';
+import { isValid } from 'date-fns';
+
+function PickDisplayDate(props) {
+  const [displayDate, setDisplayDate] = useState(props.displayDate);
+
+  useEffect(() => {
+    setDisplayDate(props.displayDate);
+  }, [props.displayDate])
+
+  useEffect(() => {
+    props.onChangeDisplayDate(displayDate);
+  }, [displayDate])
+
+  return (
+    <MuiPickersUtilsProvider utils={DateFnsUtils} id="timeline">
+      <KeyboardDatePicker style={{    
```

Comment:
you could move style into the next line so the indentation isn't so out of place

Reply:
Yes!